src/byte/lib/utils.py

```diff
@@ -22,45 +25,122 @@
     from discord.ext.commands import Context
     from discord.ext.commands._types import Check
 
+__all__ = (
+    "BaseRuffRule",
+    "RuffRule",
+    "FormattedRuffRule",
+    "PEP",
+    "is_byte_dev",
+    "linker",
+    "mention_user",
+    "mention_user_nickname",
+    "mention_channel",
+    "mention_role",
+    "mention_slash_command",
+    "mention_custom_emoji",
+    "mention_custom_emoji_animated",
+    "mention_timestamp",
+    "mention_guild_navigation",
+    "format_ruff_rule",
+    "query_all_ruff_rules",
+    "run_ruff_format",
+    "paste",
+    "chunk_sequence",
+    "query_all_peps",
+)
+
 _T = TypeVar("_T")
 
 
 class BaseRuffRule(TypedDict):
```

Comment:
I dont really like all of these being here.
idk if better spot, seems like we could have "even more files" to shorten this one.. but that also seems bad